### ISSUE TITLE
Color changed in rsvg_png

### ISSUE THREAD

**Issue by zhangnan0107** [ropensci/rsvg#46]
Hi, 

Thanks for developing the convenient tool. I would like to ask is it possible to keep the colors when converting svg to png (or other formats). I tried with one svg file where I used pink for data tracks, however, it became blue after conversion.

Thank a lot,
Nan

**Comment by jeroen** [comment]
Can you include example code and file please.

**Comment by zhangnan0107** [comment]
Sure.

Here is the svg plot I wanted to convert:
<img width="277" alt="image" src="https://github.com/user-attachments/assets/4f659275-390a-41f2-8f92-6f5029eacfd2">

Here is the png file after conversion:
<img width="319" alt="image" src="https://github.com/user-attachments/assets/0d53d7b7-1f45-4f10-a116-ae03fc3f5942">

Here is the code I used:

`svg_file <- "igv.svg"`

`png_file <- "igv.png"`

`rsvg_png(svg_file,png_file)`

Let me know if you need the content within the svg file. Thanks a lot!

**Comment by jeroen** [comment]
Yes we need the svg file. Does the same problem happen with `magick::image_read()` ?

**Comment by zhangnan0107** [comment]
Here is the svg file, and yes, I tried once last week with `magick::image_read()` to save in tiff format and the color also changed.

![PMD_ref_igv](https://github.com/user-attachments/assets/5bdcb2d7-fb9d-4de7-8a08-e4b32fc852c2)

Let me know if the file is not attached successfully : )

**Comment by jeroen** [comment]
It works fine for me on Windows and MacOS with rsvg 2.7.0. Which operating system do you use?

If you are on Linux you may need to upgrade your system.
